packages/react-component-library/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
This should be in the top level, I had to do it for dist and build ealier.

Reply:
It didn't work. When I merged your branch in it tried to commit those folders, hence me adding the gitignore.